### PR TITLE
Show user POPI info

### PR DESCRIPTION
**Pull request by rudigiesler**
This allows the user to see their personal stored data on the RapidPro POPI line

### CODE REVIEW

File: src/ussd_popi_rapidpro.js [hunk 1]
```diff
@@ -1,27 +1,173 @@
...
     var MenuState = vumigo.states.MenuState;
+    var PaginatedState = vumigo.states.PaginatedState;
 
     var GoNDOH = App.extend(function(self) {
         App.call(self, "state_start");
         var $ = self.$;
 
         self.init = function() {
+            self.rapidpro = new go.RapidPro(
+                new JsonApi(self.im, {headers: {'User-Agent': ["Jsbox/NDoH-Public"]}}),
```
Comment (OMosimege) [inline src/ussd_popi_rapidpro.js]: does it use the public line, or have its own line?

**Review by OMosimege — APPROVED** [review]
approved👍 but added one question

File: src/ussd_popi_rapidpro.js (hunk 1 above)
Comment (rudigiesler) [reply src/ussd_popi_rapidpro.js]: Nope, copy and paste error, good catch :+1:

**Comment by rudigiesler** [comment]
@OMosimege Fixed. Ready for rereview

**Pull request merged**